aws/resource_aws_ssm_maintenance_window_task.go

```diff
@@ -73,6 +74,20 @@ func resourceAwsSsmMaintenanceWindowTask() *schema.Resource {
 				},
 			},
 
+			"name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
```

Comment:
Is this really forcing a new resource? The API seems to tell the opposite (i.e. it's updatable): https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_UpdateMaintenanceWindow.html